src/components/Flex/Flex.jsx

```diff
@@ -29,7 +31,11 @@ const Flex = forwardRef(
   ) => {
     const componentRef = useRef(null);
     const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
-    const overrideStyle = useMemo(() => ({ ...style, gap: `${gap}px` }), [style, gap]);
+    const overrideStyle = useMemo(() => {
+      const overrideGapRow = gapRow || gap;
```

Comment:
but the default value is gaps.none which different from undefined. it will override the gap value even if it is exist. I think that gapRow and gapColumn default value should be undefined for fixing this.

Reply:
or maybe its working because none = 0 which in condition same ad false?